examples/standalone/model_creation/model_creation.cc

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <chrono>
+#include <iostream>
+
+#include <ignition/gazebo/components/Light.hh>
+#include <ignition/gazebo/components/LightCmd.hh>
+#include <ignition/msgs.hh>
+#include <ignition/transport.hh>
+
+using namespace std::chrono_literals;
+
+// Create a transport node.
+ignition::transport::Node node;
+
+// timeout used for services
+constexpr unsigned int timeout = 5000;
+
+void createLight()
+{
+  bool result;
+  ignition::msgs::Boolean rep;
+//! [create light]
+  ignition::msgs::EntityFactory entityFactoryRequest;
+
+  entityFactoryRequest.mutable_light()->set_name("point");
+  entityFactoryRequest.mutable_light()->set_id(22);
+  entityFactoryRequest.mutable_light()->set_parent_id(1);
+  entityFactoryRequest.mutable_light()->set_range(4);
+  entityFactoryRequest.mutable_light()->set_attenuation_linear(0.5);
+  entityFactoryRequest.mutable_light()->set_attenuation_constant(0.2);
+  entityFactoryRequest.mutable_light()->set_attenuation_quadratic(0.01);
+  entityFactoryRequest.mutable_light()->set_cast_shadows(false);
+  entityFactoryRequest.mutable_light()->set_type(ignition::msgs::Light::POINT);
+  ignition::msgs::Set(
+    entityFactoryRequest.mutable_light()->mutable_direction(),
+    ignition::math::Vector3d(0.5, 0.2, -0.9));
+  ignition::msgs::Set(entityFactoryRequest.mutable_light()->mutable_pose(),
+    ignition::math::Pose3d(0.0, 0, 3.0, 0.0, 0.0, 0.0));
+//! [create light]
+
+//! [call service create]
+  bool executedEntityFactory = node.Request("/world/empty/create",
+        entityFactoryRequest, timeout, rep, result);
+  if (executedEntityFactory)
+  {
+    if (result)
+      std::cout << "Light was created : [" << rep.data() << "]" << std::endl;
+    else
+    {
+      std::cout << "Service call failed" << std::endl;
+      return;
+    }
+  }
+  else
+    std::cerr << "Service call timed out" << std::endl;
+//! [call service create]
+}
+
+void createModelFromStr(const std::string modelStr)
```

Comment:
```suggestion
void createEntityFromStr(const std::string modelStr)
```

This also works for lights, so the name can be more generic